concord-server/server/README.md

```diff
@@ -9,40 +9,40 @@ curl -v -X POST http://127.0.0.1:9000/api/sessions --header "Content-Type: appli
 
 ### Get next phrase to label
 ```
-curl -v -X GET http://127.0.0.1:9000/api/phrases/next --header "Authorization: Bearer XXXXX"
+curl -v -X GET http://127.0.0.1:9000/api/phrases/next --header "Authorization: Bearer XXXXXX"
```

Comment:
Why the extra X?

Reply:
was inconsistent with the comment about replacing 'XXXXXX'